### PR TITLE
feat: Add gh- and npm-username opts

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/semantic-release/semantic-release/issues/434

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ semantic-release-cli setup
 	  --[no-]keychain     Use keychain to get passwords [default: true].
 	  --ask-for-passwords Ask for the passwords even if passwords are stored [default: false].
 	  --tag=<String>      npm tag to install [default: 'latest'].
+    --gh-token=<String>  GitHub auth token
+    --npm-token=<String> npm auth
```
Comment (gr2m) [inline README.md]: npm auth _token_?

Comment (gr2m) [reply README.md]: let me fix that for ya, it’s obviously a type :)

**Comment by RichardLitt** [comment]
Thanks! :)

**Comment by vanduynslagerp** [comment]
It seems this PR was merged with a linting error. See https://travis-ci.org/semantic-release/cli/jobs/276660226#L505

It doesn't prevent the code to work but it will fail any new PR and prevent `semantic-release` and `greenkeeper` to work properly.

/cc @gr2m

**Comment by gr2m** [comment]
Sorry that I didn’t catch that! Thanks @RichardLitt for sending a fix already <3

**Pull request merged**